docs/example/tests/test_config.py

```diff
@@ -39,7 +33,12 @@
         weight=1,
         obj_name="main",
         sub_module="hello_user",
-        ref_module="tests.reference",
+        ref_module="reference",
```

Comment:
Are we sure this works when we run unittest from the 'example' directory?

Reply:
No, it did not work. Fixed in latest commit. I will create a github action that also checks if this works properly.